src/pages/publication/references_tab/components/InstitutionSearch.tsx

```diff
@@ -55,6 +59,9 @@ const InstitutionSearch: FC<InstitutionSearchProps> = ({
       setValueFunction={setValueFunction}
       label={label}
       placeholder={placeholder}
+      displaySelection={true}
+      disabled={disabled || false}
```

Comment:
tror det holder å skrive `disabled={disabled}` (?) for hvis disabled-inputpropen er undefined, så vil vel disabled bli false? du må kanskje skrive `disabled={!!disabled}` da (?)